### PR TITLE
Improved documentation

### PR DESCRIPTION
**Pull request by richelbilderbeek**
I documented more, added an FAQ and analysed the retrospective.

### CODE REVIEW

File: doc/20211215_retrospective.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Also, some people want to be excluded to work on their own once in a while.
+
+ * Get more people in the team
+
+Hard to achieve, I would be open to more suggestions.
+
+ * Monthly C++ core guideline - How tos, max 5 mins
+
+I added this to the schedule as a Montryly event on Discord.
```
Comment (TheoPannetier) [inline doc/20211215_retrospective.md:39]:
```suggestion
I added this to the schedule as a Monthly event on Discord.
```

File: doc/20211215_retrospective.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Drawback (or isn't it?) is that the team gets to know one another slower;
+they only meet the people with around the same number of commits.
+
+Also, some people want to be excluded to work on their own once in a while.
+
+ * Get more people in the team
+
+Hard to achieve, I would be open to more suggestions.
+
+ * Monthly C++ core guideline - How tos, max 5 mins
```
Comment (TheoPannetier) [inline doc/20211215_retrospective.md:37]:
```suggestion
 * Monthly C++ core guideline - How to's, max 5 mins
```

File: faq.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+## I am a junior developer, what should I do?
+
+Junior developers are those that feel comfortable enough with the workflow
+to write their own tests. Next to programming, juniors will
+occasionally teach newbies.
+
+You will be practicing:
+
+ * Team development workflow
+ * git an GitHub
```
Comment (TheoPannetier) [inline faq.md:113]:
```suggestion
 * git and GitHub
```

File: faq.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+ * C++ and SFML entry level development
+
+Start from an idea of a visible change in the game,
+for example, 'The players should be able to do a little dance'.
+Create an Issue for that and assign yourself to it
+Work from big to small, in TDD fashion.
```
Comment (TheoPannetier) [inline faq.md:119]:
```suggestion
Work from big to small, in TDD ('Test-Driven Development') fashion.
```

**Review by TheoPannetier — APPROVED** [review]
Awesome, thanks for tidying the FAQ and noting down outcomes from the Retrospective!
I just caught a few typos :)

**Comment by richelbilderbeek** [comment]
@TheoPannetier thanks for fixing the typo's, I accepted all of 'em :-)

**Comment by richelbilderbeek** [comment]
@TheoPannetier it is just a formality, but hey, it is the way we work: could you merge this Pull Request? It makes you responsible for the damage it may cause :-)

**Pull request merged**